Started by timer
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Still waiting to schedule task
Waiting for next available executor on â€˜[8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99â€™
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/test-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.6.4"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/","vm_image_conf":{"image1":"rhel.img","image2":"rhel2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.112.97.62","pf1":"ens2f0","os":"RHEL"},"pkg_generator":{"host":"10.112.97.60","pf1":"ens2f1"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-2.1.1
collected 20 items / 19 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-05-05 02:54:55 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-05-05/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img]_02_54_55/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img].log, case_logdir: /root/testconfig/testlogs/2022-05-05/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img]_02_54_55
2022-05-05 02:54:55 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img] case ============================
2022-05-05 02:54:55 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-05-05 02:54:55 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-05-05 02:54:55 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-05-05 02:54:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (publickey) failed.
2022-05-05 02:54:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-05-05 02:54:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.62
2022-05-05 02:54:56 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-05-05 02:54:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (publickey) failed.
2022-05-05 02:54:56 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-05-05 02:54:56 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.112.97.60
2022-05-05 02:54:56 - test_network_sriov_stress.py - [line:64] - root - INFO - clean up iperf and taskset process...
2022-05-05 02:54:59 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-05-05 02:54:59 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:55:07 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-05-05 02:55:07 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:55:15 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-05-05 02:55:15 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:55:23 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-05-05 02:55:23 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:55:28 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 3] Opened sftp connection (server version 3)
2022-05-05 02:55:28 - sshInstance.py - [line:201] - root - INFO - start to rmmod irdma....
2022-05-05 02:55:31 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-05-05 02:55:31 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-05-05 02:55:36 - sshInstance.py - [line:204] - root - INFO - finished rmmod irdma....
2022-05-05 02:55:36 - sshInstance.py - [line:205] - root - INFO - start to rmmod ice....
2022-05-05 02:55:39 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-05-05 02:55:44 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:55:49 - sshInstance.py - [line:208] - root - INFO - finished rmmod ice....
2022-05-05 02:55:49 - sshInstance.py - [line:210] - root - INFO - start to modprobe ice....
2022-05-05 02:55:52 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-05-05 02:55:57 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:56:02 - sshInstance.py - [line:212] - root - INFO - finished modprobe ice....
2022-05-05 02:56:05 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens2f0
2022-05-05 02:56:05 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 2.50 0x80006a16 1.2852.0
expansion-rom-version: 
bus-info: 0000:4b:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-05-05 02:56:10 - test_network_sriov_stress.py - [line:71] - root - INFO - try to get ipv6 for ens2f0
2022-05-05 02:56:10 - test_network_sriov_stress.py - [line:73] - root - INFO - try to get ipv6 of pf for redhat...
2022-05-05 02:56:10 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f0:
2022-05-05 02:56:16 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Thu May  5 02:56:12 2022 from 10.239.182.99
[root@WW25ICX2 ~]# nmcli connection delete ens2f0
Connection 'ens2f0' (35e9e3d5-8ec6-48e8-9791-ae8ed169719a) successfully deleted.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 02:56:19 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-05 02:56:25 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 02:56:28 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f0:
2022-05-05 02:56:34 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f0 ifname ens22f0
Connection 'ens2f0' (49b2d14c-489a-463e-976e-171424320fdb) successfully added.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 02:56:37 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-05 02:56:43 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 02:56:46 - test_network_sriov_stress.py - [line:80] - root - INFO - try to get ipv6 for ens2f1
2022-05-05 02:56:46 - test_network_sriov_stress.py - [line:82] - root - INFO - getting ipv6 in redhat.
2022-05-05 02:56:46 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f1:
2022-05-05 02:56:52 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Thu May  5 02:47:28 2022 from 10.239.182.99
[root@localhost ~]# nmcli connection delete ens2f1
Connection 'ens2f1' (3b30579a-7ba2-444c-8b63-feac50e9ee2d) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 02:56:55 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-05 02:57:01 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 02:57:04 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f1:
2022-05-05 02:57:10 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f1 ifname enss2f1
Connection 'ens2f1' (2753e853-f63b-40f8-ae5b-d604264a1995) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 02:57:13 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-05 02:57:19 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 02:57:22 - test_network_sriov_stress.py - [line:90] - root - INFO - trying to get ipv6 addr for pf1
2022-05-05 02:57:25 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f0
2022-05-05 02:57:25 - sshInstance.py - [line:54] - root - INFO - ens2f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::40a:5e19:d320:3e25  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:9c:f7:a4  txqueuelen 1000  (Ethernet)
        RX packets 10  bytes 3037 (2.9 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 47  bytes 8073 (7.8 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-05-05 02:57:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens2f1
2022-05-05 02:57:33 - sshInstance.py - [line:54] - root - INFO - ens2f1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::fe66:4734:de0f:5b26  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a5:9b:45  txqueuelen 1000  (Ethernet)
        RX packets 36219552609  bytes 54834086480483 (49.8 TiB)
        RX errors 0  dropped 646  overruns 0  frame 0
        TX packets 1309242477  bytes 94465421232 (87.9 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-05-05 02:57:38 - test_network_sriov_stress.py - [line:621] - root - INFO - 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

2022-05-05 02:57:41 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -s | grep ens2f0 | awk '{print $1;}'
2022-05-05 02:57:41 - sshInstance.py - [line:54] - root - INFO - 0000:4b:00.0

2022-05-05 02:57:46 - test_network_sriov_stress.py - [line:156] - root - INFO - pf_id was found: 0000:4b:00.0

2022-05-05 02:57:46 - test_network_sriov_stress.py - [line:629] - root - INFO - format_pf_id is 0000\:4b\:00.0

2022-05-05 02:57:46 - test_network_sriov_stress.py - [line:632] - root - INFO - pf_id_key: 4b
2022-05-05 02:57:46 - test_network_sriov_stress.py - [line:636] - root - INFO - start to assign ip for ens2f0 of SUT1: 192.168.10.10
2022-05-05 02:57:52 - sshInstance.py - [line:99] - root - INFO - ifconfig ens2f0 192.168.10.10
[root@WW25ICX2 ~]# 
2022-05-05 02:57:55 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens2f0
2022-05-05 02:57:55 - test_network_sriov_stress.py - [line:638] - root - INFO - start to assign ip for ens2f1 of SUT2: 192.168.10.11
2022-05-05 02:58:01 - sshInstance.py - [line:99] - root - INFO - ifconfig ens2f1 192.168.10.11
[root@localhost ~]# 
2022-05-05 02:58:04 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens2f1
2022-05-05 02:58:04 - test_network_sriov_stress.py - [line:162] - root - INFO - virtulize sut1 pf to 4 vfs
2022-05-05 02:58:07 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-05-05 02:58:08 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:58:13 - test_network_sriov_stress.py - [line:171] - root - INFO - Successfully Virtualized PF to VFs.
2022-05-05 02:58:16 - sshInstance.py - [line:50] - root - INFO - lspci | grep "Virtual" |grep 4b | awk '{print $1;}'list
2022-05-05 02:58:16 - sshInstance.py - [line:54] - root - INFO - 4b:01.0
4b:01.1
4b:01.2
4b:01.3

2022-05-05 02:58:21 - test_network_sriov_stress.py - [line:185] - root - INFO - Succssfully Gotten vf_id list:

2022-05-05 02:58:24 - sshInstance.py - [line:50] - root - INFO - modprobe vfio-pci

2022-05-05 02:58:24 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:58:29 - test_network_sriov_stress.py - [line:205] - root - INFO - start to bind drive in sut1
2022-05-05 02:58:32 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.0
2022-05-05 02:58:33 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:58:38 - test_network_sriov_stress.py - [line:219] - root - INFO - Successfuly bound VF to vfio-pci
2022-05-05 02:58:41 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-stable-20.11.1/usertools/dpdk-devbind.py -b vfio-pci 4b:01.1
2022-05-05 02:58:42 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 02:58:47 - test_network_sriov_stress.py - [line:219] - root - INFO - Successfuly bound VF to vfio-pci
2022-05-05 02:58:47 - test_network_sriov_stress.py - [line:656] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-05-05 02:58:50 - sshInstance.py - [line:50] - root - INFO - lspci -Dnn | grep 4b:01.1
2022-05-05 02:58:50 - sshInstance.py - [line:54] - root - INFO - 0000:4b:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-05-05 02:58:55 - test_network_sriov_stress.py - [line:668] - root - INFO - Successfully get vf device id: 8086:1889
2022-05-05 02:58:55 - test_network_sriov_stress.py - [line:242] - root - INFO - get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
2022-05-05 02:58:58 - sshInstance.py - [line:50] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-05-05 02:58:58 - sshInstance.py - [line:54] - root - INFO - yes

2022-05-05 02:59:03 - test_network_sriov_stress.py - [line:245] - root - INFO - Default network is active? yes

2022-05-05 02:59:03 - test_network_sriov_stress.py - [line:681] - root - INFO - start to create VMs....
2022-05-05 02:59:03 - test_network_sriov_stress.py - [line:275] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/image/rhel2.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-05-05 02:59:07 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/rhel2.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.00.0,port=5900 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo rhel8.2
[root@WW25ICX2 ~]# 
2022-05-05 02:59:10 - test_network_sriov_stress.py - [line:278] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/image/rhel2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-05-05 02:59:14 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_0 --boot hd --disk path=/home/iimage/rhel2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5900 --nooautoconsole
2022-05-05 02:59:15 - sshInstance.py - [line:75] - root - INFO - 
** (process:48416): [1;33mWARNING[0m **: [34m03:00:44.358[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-05-05 02:59:17 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-05-05 02:59:20 - test_network_sriov_stress.py - [line:292] - root - INFO - Successfully created a VM: VMtest1
2022-05-05 02:59:20 - test_network_sriov_stress.py - [line:275] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/image/rhel.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-05-05 02:59:24 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/rhel.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0..0,port=5901 --noautoconsole
usage: virt-install --name NAME --memory MB STORAGE INSTALL [options]
virt-install: error: unrecognized arguments: --osinfo rhel8.2
[root@WW25ICX2 ~]# 
2022-05-05 02:59:27 - test_network_sriov_stress.py - [line:278] - root - INFO - cmd1 is not working, try to create VM with the cmd2: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/image/rhel.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-05-05 02:59:31 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 ---network network=default --hostdev pci_0000_4b_01_1 --boot hd --disk path=/home/iimage/rhel.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=5901 --noaautoconsole

** (process:48472): [1;33mWARNING[0m **: [34m03:01:01.196[0m: Entity http://pcisig.com/pci/1B36/0100 referenced but not defined
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-05-05 02:59:33 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@WW25ICX2 ~]# 
2022-05-05 02:59:36 - test_network_sriov_stress.py - [line:292] - root - INFO - Successfully created a VM: VMtest2
2022-05-05 03:00:09 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-05-05 03:00:09 - sshInstance.py - [line:54] - root - INFO - 192.168.122.185/24

2022-05-05 03:00:14 - test_network_sriov_stress.py - [line:308] - root - INFO - Successfully got VMtest1 IP: 192.168.122.185/24

2022-05-05 03:00:17 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-05-05 03:00:17 - sshInstance.py - [line:54] - root - INFO - 192.168.122.14/24

2022-05-05 03:00:22 - test_network_sriov_stress.py - [line:308] - root - INFO - Successfully got VMtest2 IP: 192.168.122.14/24

2022-05-05 03:00:22 - test_network_sriov_stress.py - [line:692] - root - INFO - ip of VM1: 192.168.122.185
2022-05-05 03:00:22 - test_network_sriov_stress.py - [line:693] - root - INFO - ip of VM1: 192.168.122.14
2022-05-05 03:00:22 - test_network_sriov_stress.py - [line:698] - root - INFO - Start ssh VM1 and conf VM1
2022-05-05 03:00:28 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.185
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-05-05 03:00:35 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.185
PING 192.168.122.185 (192.168.122.185) 56(84) bytes of data.
64 bytes from 192.168.122.185: icmp_seq=1 ttl=64 time=0.155 ms
2022-05-05 03:00:36 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=2 ttl=64 time=0.212 ms
2022-05-05 03:00:37 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=3 ttl=64 time=0.206 ms
2022-05-05 03:00:38 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=4 ttl=64 time=0.196 ms
2022-05-05 03:00:39 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=5 ttl=64 time=0.197 ms

--- 192.168.122.185 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4093ms
rtt min/avg/max/mdev = 0.155/0.193/0.212/0.021 ms
[root@WW25ICX2 ~]# 
2022-05-05 03:00:48 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.185
root@192.168.122.185's password: 
2022-05-05 03:00:57 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Tue May  3 20:58:19 2022 from 192.168.122.1
[root@localhost ~]# 
2022-05-05 03:01:00 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-05-05 03:01:06 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}}'
0000:00:05.0
[root@localhost ~]# 
2022-05-05 03:01:15 - test_network_sriov_stress.py - [line:414] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-05-05 03:01:21 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
[root@localhost ~]# 
2022-05-05 03:01:26 - test_network_sriov_stress.py - [line:428] - root - INFO - Successfully got vf device name of vm: ens5
2022-05-05 03:01:26 - test_network_sriov_stress.py - [line:440] - root - INFO - try to get ipv6 for ens5
2022-05-05 03:01:26 - test_network_sriov_stress.py - [line:442] - root - INFO - try to get ipv6 in redhat
2022-05-05 03:01:26 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens5:
2022-05-05 03:01:32 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens5
Connection 'ens5' (448318ad-8364-45c6-8d8a-c16c7af52241) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:01:35 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:01:41 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:01:44 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens5:
2022-05-05 03:01:50 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens5 ifname ens5 [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            5
Connection 'ens5' (46b37fae-cafd-4600-b996-1cd5f8606a19) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:01:53 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:01:59 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:02:08 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::e0f9:861e:1392:1118  prefixlen 64  scopeid 0x20<link>
        ether 8a:53:cd:e7:68:3a  txqueuelen 1000  (Ethernet)
        RX packets 32  bytes 7187 (7.0 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 43  bytes 6443 (6.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
[root@localhost ~]# 
2022-05-05 03:02:11 - test_network_sriov_stress.py - [line:455] - root - INFO - assign ip address for VM...
2022-05-05 03:02:17 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.12
[root@localhost ~]# 
2022-05-05 03:02:20 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-05-05 03:02:26 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.185 closed.
[root@WW25ICX2 ~]# 
2022-05-05 03:02:29 - test_network_sriov_stress.py - [line:704] - root - INFO - Start ssh VM2 and conf VM2
2022-05-05 03:02:35 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.14
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-05-05 03:02:42 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.14
PING 192.168.122.14 (192.168.122.14) 56(84) bytes of data.
64 bytes from 192.168.122.14: icmp_seq=1 ttl=64 time=0.305 ms
2022-05-05 03:02:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.14: icmp_seq=2 ttl=64 time=0.239 ms
2022-05-05 03:02:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.14: icmp_seq=3 ttl=64 time=0.217 ms
2022-05-05 03:02:45 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.14: icmp_seq=4 ttl=64 time=0.237 ms
2022-05-05 03:02:46 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.14: icmp_seq=5 ttl=64 time=0.204 ms

--- 192.168.122.14 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4114ms
rtt min/avg/max/mdev = 0.204/0.240/0.305/0.037 ms
[root@WW25ICX2 ~]# 
2022-05-05 03:02:55 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.14
root@192.168.122.14's password: 
2022-05-05 03:03:04 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Tue May  3 20:56:12 2022 from 192.168.122.1
[root@localhost ~]# 
2022-05-05 03:03:07 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-05-05 03:03:13 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}}'
0000:00:05.0
[root@localhost ~]# 
2022-05-05 03:03:22 - test_network_sriov_stress.py - [line:414] - root - INFO - successfully got vf pci id of vm: 0000:00:05.0
2022-05-05 03:03:28 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
[root@localhost ~]# 
2022-05-05 03:03:33 - test_network_sriov_stress.py - [line:428] - root - INFO - Successfully got vf device name of vm: ens5
2022-05-05 03:03:33 - test_network_sriov_stress.py - [line:440] - root - INFO - try to get ipv6 for ens5
2022-05-05 03:03:33 - test_network_sriov_stress.py - [line:442] - root - INFO - try to get ipv6 in redhat
2022-05-05 03:03:33 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens5:
2022-05-05 03:03:39 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens5
Connection 'ens5' (916de023-d8ef-4e9e-afb9-6237d90e860d) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:03:42 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:03:49 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:03:52 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens5:
2022-05-05 03:03:58 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens5 ifname ens5 [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            5
Connection 'ens5' (aff7d53c-2bef-4f91-a2ba-43e920aee5b7) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:04:01 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:04:07 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:04:16 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::87a0:6f07:da65:3482  prefixlen 64  scopeid 0x20<link>
        ether b6:80:ab:8e:24:11  txqueuelen 1000  (Ethernet)
        RX packets 30  bytes 7847 (7.6 KiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 63  bytes 11274 (11.0 KiB)
        TX errors 3  dropped 0 overruns 0  carrier 0  collisions 0
[root@localhost ~]# 
2022-05-05 03:04:19 - test_network_sriov_stress.py - [line:455] - root - INFO - assign ip address for VM...
2022-05-05 03:04:25 - sshInstance.py - [line:99] - root - INFO - ifconfig ens5 192.168.10.13
[root@localhost ~]# 
2022-05-05 03:04:28 - test_network_sriov_stress.py - [line:365] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-05-05 03:04:34 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.14 closed.
[root@WW25ICX2 ~]# 
2022-05-05 03:04:37 - test_network_sriov_stress.py - [line:712] - root - INFO - Ping two vms IP from SUT:
2022-05-05 03:04:41 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.201 ms
2022-05-05 03:04:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.104 ms
2022-05-05 03:04:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.100 ms
2022-05-05 03:04:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.120 ms
2022-05-05 03:04:45 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.153 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4120ms
rtt min/avg/max/mdev = 0.100/0.135/0.201/0.039 ms
[root@WW25ICX2 ~]# 
2022-05-05 03:04:52 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.187 ms
2022-05-05 03:04:53 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.120 ms
2022-05-05 03:04:54 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.126 ms
2022-05-05 03:04:55 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.122 ms
2022-05-05 03:04:56 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.122 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4115ms
rtt min/avg/max/mdev = 0.120/0.135/0.187/0.027 ms
[root@WW25ICX2 ~]# 
2022-05-05 03:04:59 - test_network_sriov_stress.py - [line:720] - root - INFO - ping VMtest2 IP from VMtest1
2022-05-05 03:05:05 - sshInstance.py - [line:99] - root - INFO - ssh-key -R 192.168.122.185
bash: ssh-key: command not found...
[root@WW25ICX2 ~]# 
2022-05-05 03:05:12 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.185
PING 192.168.122.185 (192.168.122.185) 56(84) bytes of data.
64 bytes from 192.168.122.185: icmp_seq=1 ttl=64 time=0.135 ms
2022-05-05 03:05:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=2 ttl=64 time=0.103 ms
2022-05-05 03:05:14 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=3 ttl=64 time=0.186 ms
2022-05-05 03:05:15 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=4 ttl=64 time=0.175 ms
2022-05-05 03:05:16 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.185: icmp_seq=5 ttl=64 time=0.186 ms

--- 192.168.122.185 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4121ms
rtt min/avg/max/mdev = 0.103/0.157/0.186/0.032 ms
[root@WW25ICX2 ~]# 
2022-05-05 03:05:25 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.185
root@192.168.122.185's password: 
2022-05-05 03:05:34 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Wed May  4 15:02:24 2022 from 192.168.122.1
[root@localhost ~]# 
2022-05-05 03:05:37 - test_network_sriov_stress.py - [line:392] - root - INFO - Successfully ssh VM.
2022-05-05 03:05:41 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.229 ms
2022-05-05 03:05:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.073 ms
2022-05-05 03:05:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.060 ms
2022-05-05 03:05:44 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.044 ms
2022-05-05 03:05:45 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.048 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 78ms
rtt min/avg/max/mdev = 0.044/0.090/0.229/0.070 ms
[root@localhost ~]# 
2022-05-05 03:05:54 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.185 closed.
[root@WW25ICX2 ~]# 
2022-05-05 03:05:57 - test_network_sriov_stress.py - [line:728] - root - INFO - ping VMtest2 IP from VMtest1
2022-05-05 03:06:01 - sshInstance.py - [line:75] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.428 ms
2022-05-05 03:06:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.142 ms
2022-05-05 03:06:03 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.268 ms
2022-05-05 03:06:04 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.096 ms
2022-05-05 03:06:05 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.259 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4085ms
rtt min/avg/max/mdev = 0.096/0.238/0.428/0.116 ms
[root@WW25ICX2 ~]# 
2022-05-05 03:06:11 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-05-05 03:06:13 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 03:06:18 - test_network_sriov_stress.py - [line:327] - root - INFO - Successfully removed VFs from PF.
2022-05-05 03:06:21 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-05-05 03:06:21 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 destroyed


2022-05-05 03:06:26 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest1
2022-05-05 03:06:29 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-05-05 03:06:29 - sshInstance.py - [line:54] - root - INFO - Domain VMtest1 has been undefined


2022-05-05 03:06:34 - test_network_sriov_stress.py - [line:350] - root - INFO - Successfully removed the VM: VMtest1
2022-05-05 03:06:37 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-05-05 03:06:37 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 destroyed


2022-05-05 03:06:42 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest2
2022-05-05 03:06:45 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-05-05 03:06:46 - sshInstance.py - [line:54] - root - INFO - Domain VMtest2 has been undefined


2022-05-05 03:06:51 - test_network_sriov_stress.py - [line:350] - root - INFO - Successfully removed the VM: VMtest2
.2022-05-05 03:06:51 - test_network_sriov_stress.py - [line:102] - root - INFO - removing VMs: VMtest1, VMtest2
2022-05-05 03:06:54 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-05-05 03:06:54 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-05-05 03:06:59 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest2
2022-05-05 03:07:02 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-05-05 03:07:02 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest2'


2022-05-05 03:07:07 - test_network_sriov_stress.py - [line:353] - root - ERROR - Failed to removed the VM VMtest2
2022-05-05 03:07:10 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-05-05 03:07:10 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-05-05 03:07:15 - test_network_sriov_stress.py - [line:340] - root - INFO - Successfully shut down the VM: VMtest1
2022-05-05 03:07:18 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-05-05 03:07:18 - sshInstance.py - [line:54] - root - INFO - error: failed to get domain 'VMtest1'


2022-05-05 03:07:23 - test_network_sriov_stress.py - [line:353] - root - ERROR - Failed to removed the VM VMtest1
2022-05-05 03:07:26 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens2f0/device/sriov_numvfs
2022-05-05 03:07:26 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 03:07:31 - test_network_sriov_stress.py - [line:107] - root - INFO - try to reset network for ens2f0
2022-05-05 03:07:31 - test_network_sriov_stress.py - [line:109] - root - INFO - try to reset network for redhat...
2022-05-05 03:07:31 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f0:
2022-05-05 03:07:37 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens2f0
Connection 'ens2f0' (49b2d14c-489a-463e-976e-171424320fdb) successfully deleted.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 03:07:40 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:07:46 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 03:07:49 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f0:
2022-05-05 03:07:55 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f0 ifname ens22f0
Connection 'ens2f0' (436d99a0-1c2f-4929-afef-46a1d8372ddc) successfully added.
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 03:07:58 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:08:04 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@WW25ICX2 ~]# 
[root@WW25ICX2 ~]# 
2022-05-05 03:08:07 - test_network_sriov_stress.py - [line:116] - root - INFO - try to reset network for ens2f1
2022-05-05 03:08:07 - test_network_sriov_stress.py - [line:118] - root - INFO - try to reset network for redhat
2022-05-05 03:08:07 - sshInstance.py - [line:188] - root - INFO - starting to delete network of ens2f1:
2022-05-05 03:08:13 - sshInstance.py - [line:99] - root - INFO - nmcli connection delete ens2f1
Connection 'ens2f1' (2753e853-f63b-40f8-ae5b-d604264a1995) successfully deleted.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:08:16 - sshInstance.py - [line:190] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:08:22 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:08:25 - sshInstance.py - [line:194] - root - INFO - starting to add network of ens2f1:
2022-05-05 03:08:31 - sshInstance.py - [line:99] - root - INFO - nmcli connection add type ethernet con-name ens2f1 ifname enss2f1
Connection 'ens2f1' (71bd9aaa-2466-4c76-a6b7-967d74932530) successfully added.
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:08:34 - sshInstance.py - [line:197] - root - INFO - restart networkManager to make setting valid
2022-05-05 03:08:40 - sshInstance.py - [line:99] - root - INFO - systemctl restart NetworkManager
[root@localhost ~]# 
[root@localhost ~]# 
2022-05-05 03:08:46 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-05-05 03:08:46 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 03:08:51 - test_network_sriov_stress.py - [line:126] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-05-05 03:08:51 - test_network_sriov_stress.py - [line:128] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-05-05/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img]_02_54_55
2022-05-05 03:08:54 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-05-05 03:08:54 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 03:08:59 - test_network_sriov_stress.py - [line:130] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-05-05 03:09:02 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/*.txt
2022-05-05 03:09:02 - sshInstance.py - [line:54] - root - INFO - 
2022-05-05 03:09:07 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-05-05 03:09:07 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel2.img-rhel.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:116
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:116: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:169
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:169: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:102
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:102: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:141
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:141: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:617
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:617: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:744
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:744: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:984
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:984: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1123
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1123: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1124
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:1124: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:119
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:119: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:156
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:156: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:157
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:157: PytestUnknownMarkWarning: Unknown pytest.mark.connect - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.connect

npxtester/Tests/testNics/test_nics.py:189
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:189: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:190
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:190: PytestUnknownMarkWarning: Unknown pytest.mark.static_ip - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.static_ip

npxtester/Tests/testNics/test_nics.py:221
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:221: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:222
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:222: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:308
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:308: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:334
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:334: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:377
  /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:377: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 19 deselected, 19 warnings in 868.53s (0:14:28) ===========
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
[8mha:////4LN/0LMxu1hyybTBeE6CHwDx82bv/lkvxFk3cdgHTSuWAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3favo5XL4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4FgvIflgycWoJq7MsZycbV9d7HENRG2WSX14oF+Me/GuAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAR2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwZyt32U/vwBq1seBO+AAAA[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P37N80+NRbzyFlga2j1WDon6H/4VmnQwrRysqQpQX/3AAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigagQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUO62n9qHN+xdYwq+AAAA[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/test-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P3ih/vRXCzaXCkS1CQDBQXrb3I4EZcCr0NGGH+tKlR4AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigUSDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgEjVhSLvgAAAA==[0m[Pipeline] }
[8mha:////4ISV0Ptmba0km8a80Gq+UClogdZnPDxCJXroOoeI5sULAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgE81EHZvgAAAA==[0m[Pipeline] // stage
[8mha:////4LmA5LOJ1yGOO5YQ6RYn+/szIsYEkfxAArsUKSFs0Xa0AAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEBCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AE/B/a+9AAAA[0m[Pipeline] }
[8mha:////4KVKO0UZMawo8C9tjXN4L1+ryWbU4zmvD2sDlsKoyYtXAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUVI5jZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL/eCrPwAXH35DL0AAAA=[0m[Pipeline] // node
[8mha:////4A8vvhfR6oegt+2XAbvn3Z0Fx6j3lb1QUh2abilCs+OGAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBAx/ra70AAAA=[0m[Pipeline] End of Pipeline
Finished: SUCCESS
